bin/bundle

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env ruby
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
```

Comment:
What's motivating this change? And should it also trigger a change to the use of `__FILE__` in `test/test_helper.rb`?

Reply:
This is new rails generated code. There is an upgrade command when you change versions and it spit this out.

Reply:
👍